modules/ROOT/pages/_partials/sync/server-realtime-updates.adoc

```diff
@@ -1,3 +1,125 @@
 = Realtime Updates
 
-TODO
\ No newline at end of file
+A core concept of GraphQL is subscriptions. These subscriptions can be leveraged to provide a mechanism for real time updates.
+To enable this on the server, an appropriate schema should be defined. A recommended implementation pattern can be seen in the schema below.
+
+```
+const { PubSub } = require('apollo-server');
+const { gql } = require('@aerogear/apollo-voyager-server')
+
+
+const pubSub = new PubSub();
+
+const typeDefs = gql `
+type Task {
+  id: ID!
+  title: String!
+  description: String!
+}
+
+type Mutation {
+  createTask(title: String!, description: String!): Task
+}
+
+type Subscription {
+  tasks: TaskSubscription!
+}
+
+type TaskSubscription {
+  action: actionType!
+  task: Task!
+}
+
+enum actionType {
+  CREATED
+  MUTATED
+  DELETED
+}
+`
+
+const resolvers = {
+    Mutation: {
+        createTask: async (obj, args, context, info) => {
+            const result = tasks.create(args)
+            pubSub.publish('tasks', {
+                tasks: {
+                    action: "CREATED",
+                    task: result
+                }
+              });
+            return result
+        }
+    },
+    Subscription: {
+        tasks: {
+            subscribe: () => pubSub.asyncIterator('tasks')
+        }
+  },
+}
+```
+
+In this example schema, we first set up a `Task` type, and then define the Mutations and Subscriptions required.
+When a new task is created, the `publish` method is called with information about the type of action performed along
+with the result of this action. The benefit of this type of subscription is that we can define one subscription channel
+`tasks` instead of `taskCreated`, `taskUpdated` etc. This reduces overhead on the client as there is only one
+subscription per object type. The client can then use the `action` field to decide what to do with subscription data.
+For documentation on how to use Subscriptions on your client,
+see xref:sync-js-client-realtime-updates[Realtime Updates]. You can then use the above schema as follows in a server.
+
+
+```
+const { VoyagerServer } = require('@aerogear/voyager-server')
```

Comment:
This code is very volatile. I will avoid putting lots of code without adding much context - it will be maintenance hell in long run. I can tell that this got outdated already.